deploy/install-driver.sh

```diff
@@ -38,6 +38,7 @@ kubectl apply -f $repo/csi-azuredisk-driver.yaml
 kubectl apply -f $repo/crd-csi-node-info.yaml
 kubectl apply -f $repo/rbac-csi-azuredisk-controller.yaml
 kubectl apply -f $repo/rbac-csi-azuredisk-node.yaml
+kubectl apply -f $repo/rbac-csi-azuredisk-secret.yaml
```

Comment:
it looks like `rbac-csi-azuredisk-secret.yaml` and `rbac-csi-azuredisk-node.yaml` are redudunt, pls fix it
@ZeroMagic